refocus-bdk-server.js

```diff
@@ -593,13 +594,13 @@ module.exports = (config) => {
 
   /**
     *
-    * @param {string} botName - Contains name of bot
+    * @param {string} name - Contains name of bot
     */
-  function startHeartBeat(botName){
+  function startHeartBeat(name){
     setInterval(() => {
       const currentTimestamp = new Date();
       const requestBody = { currentTimestamp };
-      genericPost(SERVER+API+BOTS_ROUTE+'/'+botName+'/heartbeat', requestBody,
+      genericPost(SERVER+API+BOTS_ROUTE+'/'+name+'/heartbeat', requestBody,
```

Comment:
Should these be template strings rather than concatenations? Just style wise

Reply:
i like concatenations more than template strings as only exactly what i put in is printed